src/ploneintranet/activitystream/browser/activity_provider.py

```diff
@@ -144,10 +136,10 @@ def text(self):
         return text
 
     def is_attachment_supported(self):
-        return IAttachmentStoragable is not None
+        return True
 
     def is_preview_supported(self):
-        return IDocconv is not None
+        return True
 
```

Comment:
Can we remove the checks for is_preview_supported and is_attachment_supported? They return True everywhere.